src/test/ajaxWrapper.test.js

```diff
@@ -119,4 +133,64 @@ describe("ajaxWrapper", function () {
 
   });
 
+  describe("on POST request", function () {
+
+    it("sends the correct payload", function (done) {
+      var response = {body: ""};
+      var request = {method: null};
+      var payload = {"key": "value"};
+
+      this.server.on("request", function (req) {
```

Comment:
That is true with the freedom.
Will can refactor it later by introducing the PUT-test and because we need this twice then. ;)